### DISCUSSION TITLE
Auto rural level bug

### DISCUSSION THREAD

**Discussion by GGyozo** [Sau1707/ModernBot#30] [General]
Hi!

I have been using your bot for quite a long time and I am very satisfied with it.
The only problem i have with it is that it doesn't pull npc villages to level 6.
When I take a new island town and start the auto rural level module, it starts, but only upgrades them to level 1 and stops. It was working normally until I upgraded the npc villages in my first 10 towns to level 6.
On new server it works normally, but there i use it directly only up to lvl5, but it would be nice if it would work up to lvl6.

Thanks in advance for all your help!

**Comment by Sau1707** [comment]
Interesting, I'm gonna take a look at this, I was also seeing strange behavior
